forge-gui/res/cardsfolder/upcoming/raven_of_fell_omens.txt

```diff
@@ -7,4 +7,4 @@ T:Mode$ CommitCrime | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ Tr
 SVar:TrigLoseLife:DB$ LoseLife | Defined$ Opponent | LifeAmount$ 1 | SubAbility$ DBGainLife
 SVar:DBGainLife:DB$ GainLife | Defined$ You | LifeAmount$ 1
 DeckHas:Ability$LifeGain
-Oracle:Flying\nWhenever you commit a crime, each opponent loses 1 life and you gain 1 life. This ability triggers only once each turn. (Targeting oppenents, anything they control, and/or cards in their graveyard is a crime.)
+Oracle:Flying\nWhenever you commit a crime, each opponent loses 1 life and you gain 1 life. This ability triggers only once each turn. (Targeting opponents, anything they control, and/or cards in their graveyard is a crime.)
```

Comment:
should be fixed in TriggerDescription too

Reply:
Done. I hope.